### ISSUE TITLE
[mxnet paper] googlenet weight decay parameter  

### ISSUE THREAD

**Issue by qiaohaijun** [dmlc/mxnet#793]
Yesterday I read the paper, and I find the weight decay is 0.05, is this correct? 
@mli

**Comment by mli** [comment]
currently i'm using eta *= 0.94 for each epoch. it's seems not optimal..

**Comment by winstywang** [comment]
you mean weight decay of learning rate?

**Comment by mli** [comment]
ah, you mean the learningsys paper. i guess that numbers are not correct. we used 0.94 for learning rate decay, 0.00001 for weight decay

the codes are  https://github.com/dmlc/mxnet/blob/master/example/image-classification/train_model.py

**Comment by qiaohaijun** [comment]
@mli yes,the paper you post yesterday from weibo

**Comment by qiaohaijun** [comment]
@winstywang the parameter's name is wd, I think it is weight decay.

**Comment by winstywang** [comment]
@qiaohaijun It should be wrong, see the clarification of mu in previous post

**Comment by qiaohaijun** [comment]
@mli @winstywang  I got it, thx all

**Comment by qiaohaijun** [comment]
@mli googlenet with bn or not?

**Comment by mli** [comment]
the numbers in the paper is fixed. 

we only tested inception-bn, the proper number for googlenet is still unknown to me yet.
